client/state/data-layer/wpcom/concierge/schedules/initial/from-api.js

```diff
@@ -0,0 +1,17 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import makeJsonSchemaParser from 'lib/make-json-schema-parser';
+import responseSchema from './schema';
+
+export const convertToDate = timestampInSeconds => timestampInSeconds * 1000;
+
+export const transform = response => {
+	return {
+		availableTimes: response.available_times.map( convertToDate ),
```

Comment:
Is there any reason to omit the data from `next_appointment` here? We'll need it to determine whether or not the customer should be allowed to take another session, so as long as we're bootstrapping the data fetch here and testing the response, it makes sense to me to include the full response we'll be using.

Reply:
Not as such, it was left out more to keep this PR a bit simpler, but I think you make a good point above, where you point out that including it will add context to future developers. I'll add it in :)